projects/arlas-components/src/lib/components/mapgl-legend/mapgl-legend.component.html

```diff
@@ -106,3 +106,24 @@
   </div>
 </div>
 
+<ng-template #legend let-legend='legend' let-titlePrefix="titlePrefix">
+  <ng-container *ngIf="(legend.title | formatLegend) as legendResult ">
+    <div class="legend-title">
+      <ng-container *ngIf="titlePrefix">{{ titlePrefix | translate }} : </ng-container>
```

Comment:
If we can isolate what possible titlePrefix there are to add to the i18n files, that would be perfect ! By the way, since the spacing before `:` is not the same in every language, I suggest you add it in the key to translate, or integrate it in the legend in another way